tests/scripts/analyze_outcomes.py

```diff
@@ -302,6 +302,39 @@ def do_analyze_driver_vs_reference(outcome_file, args):
                     # case above.
                     ('Key ASN1 (OneAsymmetricKey X25519, doesn\'t match masking '
                      'requirements, from RFC8410 Appendix A but made into version 0)'),
+                    # Compressed points and SpecifiedECDomain curves require ECP_LIGHT.
```

Comment:
Note: again, this should be simplified by #7442, since we'll be able to remove `MBEDTLS_PK_PARSE_EC_COMPRESSED` and `MBEDTLS_PK_PARSE_EC_EXTENDED` from the reference component. (Again, not change requested here.)